bundle/manifests/lvms-operator-network-policy_networking.k8s.io_v1_networkpolicy.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  labels:
+    app.kubernetes.io/component: network-policy
+    app.kubernetes.io/name: lvms-operator-network-policy
+    app.kubernetes.io/part-of: lvms-provisioner
+  name: lvms-operator-network-policy
+spec:
+  egress:
+  - ports:
+    - port: 6443
+      protocol: TCP
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          kubernetes.io/metadata.name: system
```

Comment:
When we deploy this through OLM, will it remain as `system`? It looks similar to what we had by using the hard-coded namespace in the service monitor.